tests/integration_tests/conftest.py

```diff
@@ -43,6 +43,15 @@
 ADMIN_SCHEMA_NAME = "admin_database"
 
 
+# When running tests it's important to disable autoflush to prevent dirty reads. If a
+# command modifies the database but doesn't explicitly commit the transaction, a
+# subsequent read won't see the data because we use READ COMMITTED isolation level. But
+# many integration tests write and read in the same session, and because by default
+# SQLAlchemy uses autoflush, the test would see data that in real world wouldn't be
+# there.
+db.session.configure(autoflush=False)
```

Comment:
QQ: would this be needed/preferred on the `tests/unit_tests/` side of the house too? If so it could be great to factor out into `tests/common/` (?)

Reply:
just that one question, otherwise PR LGTM